AK/QuickSort.h

```diff
@@ -113,8 +113,7 @@ void single_pivot_quick_sort(Iterator start, Iterator end, LessThan less_than)
             return;
 
         int pivot_point = size / 2;
-        if (pivot_point)
-            swap(*(start + pivot_point), *start);
+        swap(*(start + pivot_point), *start);
```

Comment:
Please explain why this is redundant (in the commit message)
At minimum it seems to save two extra iterator accesses, so it's not _completely_ redundant.

Reply:
Fair enough, please put that reasoning in the commit message :)